lstchain/reco/dl0_to_dl1.py

```diff
@@ -182,13 +188,24 @@ def r0_to_dl1(input_filename = get_dataset_path('gamma_test_large.simtel.gz'),
     is_simu = source.metadata['is_simulation']
 
     source.allowed_tels = config["allowed_tels"]
-    source.max_events = config["max_events"]
+    source.max_events = config["max_events"]+1
 
     metadata = global_metadata(source)
     write_metadata(metadata, output_filename)
 
-    cal = load_calibrator_from_config(config)
+    cal_mc = load_calibrator_from_config(config)
+
+    
+    # Camera geometry TBD: read from config file?
+    geom = load_camera_geometry()
 
+    # Dictionary to store muon ring parameters
+    muon_parameters  = create_muon_table()
+
+    plot_calib.read_file(calibration_path)
+    bad_pixels = plot_calib.calib_data.unusable_pixels[0]
+    print(f"Found a total of {np.sum(bad_pixels)} bad pixels.")
```

Comment:
Hi @moralejo,
you do not need to read the calibration file. You find the unusable pixels in each event monitoring container:
event.mon.tel[telid].calibration.unusable_pixels

Also, bad pixels are now getting gain=0, so you do not need anymore to  correct for them

Reply:
Sorry, I realise here you are at DL1 level, my first comment is not correct. You do not have the monitoring data here.
But the second comment is correct, you should not more need it...

Reply:
> Hi @moralejo,
> you do not need to read the calibration file. You find the unusable pixels in each event monitoring container:
> event.mon.tel[telid].calibration.unusable_pixels

ok, good, thanks!

> Also, bad pixels are now getting gain=0, so you do not need anymore to correct for them

Ok, unless one does the muon analysis with pre-existing DL1 files in which this was not yet the case. I guess I can keep it with a warning to eventually remove it.